app.R

```diff
@@ -0,0 +1,1268 @@
+library(boastUtils) 
+# test
+
+# Load additional dependencies and setup functions
+source("chisqplot.R") # Loads the script to make the chi sq plot.
+
+# Define UI for App ----
+ui <- list(
+  ## Create the app page ----
+  dashboardPage(
+    skin = "yellow",
+    ### Create the app header ----
+    dashboardHeader(
+      title = "Goodness-of-Fit Test",
+      titleWidth = 250,
+      tags$li(
+        class = "dropdown", 
+        actionLink("info", 
+                   icon("info"))
+        ),
+      tags$li(
+        class = "dropdown",
+        boastUtils::surveyLink(
+          name = "Chi-Square Goodness-of-Fit-Test and Simulation"
+        )
+      ),
+      tags$li(
+        class = "dropdown",
+        tags$a(href = 'https://shinyapps.science.psu.edu/', 
+               icon("home"))
+      )
+    ),
+    ### Create the sidebar/left navigation menu ----
+    dashboardSidebar(
+      sidebarMenu(
+        id = "pages",
+        menuItem("Overview", tabName = "overview", icon = icon("dashboard")),
+        menuItem("Prerequisites", tabName = "prerequisites", icon = icon("book")),
+        menuItem("Example", tabName = "example", icon = icon("wpexplorer")),
+        menuItem("Explore", tabName = "explore", icon = icon("wpexplorer")),
+        menuItem("References", tabName = "references", icon = icon("leanpub"))
+      ),
+      tags$div(
+        class = "sidebar-logo",
+        boastUtils::sidebarFooter()
+      )
+    ),
+    ### Create the content ----
+    dashboardBody(
+      tabItems(
+        #### Set up the Overview Page ----
+        tabItem(
+          tabName = "overview",
+          withMathJax(),
+          h1("Chi-Square Goodness-fit-Test"),
+          # Please reformat long lines of code; the line below is an example
+          p("In this app you will explore the Chi-Square Goodness-fit-Test with
+            simulations. The test is applied when you have categorical variables
+            from a population."),
+          br(),
+          h2("Instructions"),
+          tags$ul(
+            tags$li("Select one of the scenarios for the proportion in each category (Equal Probabilities or Different Probabilities)."),
+            tags$li("Move the sliders to change the values of number of observations, number of categories, and number of simulations."),
+            tags$li("A p-value is calculated and plotted for each simulation. You can click a p-value on the plot to see the summary table for that dataset."),
+            tags$li("When there are more than 50 simulations, only a histogram of p-values is shown."),
+            tags$li("You can use the Explore page to explore your own data and null hypothesis.")
+          ),
+          br(),
+          ##### Go Button ----
+          div(
+            style = "text-align: center",
+            bsButton(inputId = "overviewBotton",
+                     label = "Prerequisites",
+                     size = "large",
+                     icon = icon("bolt"),
```

Comment:
Change icon to prerequisites icon instead of bolt

Reply:
Hi Shravani,
Are u trying to say changing the "inputId" or "icon"?